orangecontrib/argument/miner/processor.py

```diff
@@ -1,79 +1,151 @@
-"""Argument processor module.
+"""Argument processor module."""
 
-This compute the usefulness of arguments that will be used for
-pre-filtering the arguments for the following mining steps.
-"""
-
-from importlib.util import find_spec
+from typing import List, Dict
 import copy
 import math
 
-import spacy
 import numpy as np
 import pandas as pd
 
 
-class ArgumentProcessor:
-    """Process argument before mining.
+def check_columns(expected_cols: List[str], data: pd.DataFrame):
+    """Check if a list of given columns exist in a given Pandas dataframe.
+
+    Args:
+        expected_cols (List[str]): list of columns to check
+        df (pd.DataFrame): pandas dataframe to check
+    """
+    missing_cols = [i for i in expected_cols if i not in data.columns]
+    assert (
+        len(missing_cols) == 0
+    ), f"Missing columns in the input dataframe: {missing_cols}."
+
+
+def _match_list_size(*args: List):
+    """With an arbitrary number of lists as input, check if they are in the same size."""
+    lens = [len(arg) for arg in args]
+    assert all(x == lens[0] for x in lens), f"Input size not match: {lens}."
```

Comment:
Personally, I think I would put this into a single line:
```suggestion
    assert all(len(list) == len(args[0]) for list in args) f"Input sizes do not match: {args}."
```
But is a matter of personal preference.